.github/workflows/build-release.yml

```diff
@@ -1,37 +1,34 @@
-name: Build Release
+name: build-release
+on: [workflow_dispatch]
 
-on: [ workflow_dispatch ]
+permissions:
+  contents: write
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
+      image: eclipse-temurin:17-jdk
+      options: --user root
     steps:
+      - run: apt update && apt install git -y && git --version
       - uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v1
         with:
-          java-version: 8
-      - name: Cache Iris
-        uses: actions/cache@v2
+          fetch-depth: 0
+      - uses: FabricMC/fabric-action-scripts@v1
+        id : changelog
```

Comment:
Is this needed? We usually manually write changelogs anyway, because that always gives us a nicer and more user-friendly result.

Reply:
Not sure, It got a bit funny when I didn't have it, but I'll try again later today

Reply:
Removed and **seems** to be working